site/src/presentation/slide.less

```
@@ -0,0 +1,39 @@
+.presentation-slide {
+  display: flex;
+  position: relative;
+  max-height: 100%;
+
+  @media print {
+    // A4 landscape height
+    // min-height: 210mm;
```

cosmetic: do we need this as a comment?
```suggestion
    // min-height: 210mm;
```